packages/backend/src/utils/captcha.ts

```diff
@@ -0,0 +1,32 @@
+import axios from 'axios'
+
+let CAPTCHA_SECRET_KEY: string
+function getSecretKey(): string {
+  if (CAPTCHA_SECRET_KEY) return CAPTCHA_SECRET_KEY
+
+  const secret = process.env.CAPTCHA_SECRET_KEY as string
+  // Check if secret key empty or still the default value
+  if (!secret || secret == 'your-google-recaptcha-secretkey') {
+    // Error message for backend log only, not for client
+    console.error(
```

Comment:
you might want to use logger that is available via import `@logger`  and call `logger.error()` from it

Reply:
Ah, was wondering how to log something at backend. thx